src/mmw/apps/modeling/views.py

```diff
@@ -494,3 +494,58 @@ def boundary_layer_detail(request, table_code, obj_id):
             return Response(geojson)
         else:
             return Response(status=status.HTTP_400_BAD_REQUEST)
+
+
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def plot_point_source(request):
+    try:
+        query = '''
+              SELECT ST_AsGeoJSON(geom) as geometry, city, state, npdes_id,
+                     mgd, kgn_yr, kgp_yr
+              FROM ms_pointsource_drb
+              UNION
+              SELECT ST_AsGeoJSON(geom) as geometry, city, state, npdes_id,
+                     mgd, kgn_yr, kgp_yr
+              FROM ms_pointsource
+              '''
+    except (KeyError, IndexError):
```

Comment:
I don't think the statement within this try/catch block can trigger these exceptions.